### PR TITLE
Settings improvements 1

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: scripts/superdesk/directives/sdCheck.js [hunk 1]
```diff
@@ -30,10 +30,19 @@ export default angular.module('superdesk.check.directives', [])
                 element.on('click', function(e) {
                     $scope.$apply(function() {
                         ngModel.$setViewValue(!ngModel.$viewValue);
+                        console.log(attrs);
```
Comment (gbbr) [inline scripts/superdesk/directives/sdCheck.js]: Is this needed?

**Comment by gbbr** [comment]
_as per Slack discussions with @pavlovicnemanja:_ 

We will try to use [`transclude`](https://docs.angularjs.org/api/ng/directive/ngTransclude) here as discussed on Slack. It will give developers more control and freedom over styling the checkbox.

Evaluating and appending user input to the DOM can many times be an open window to XSS attacks, so we should be a bit more weary when considering to use them. They are considered advanced operations that are for extremely rare use-cases. Most common operations can easily be resolved with standard angular functionalities.

**Comment by pavlovicnemanja** [comment]
👍

**Comment by Aca-jov** [comment]
:+1:

**Pull request merged**